### DISCUSSION TITLE
Does WebdriverIO has grouping test cases concept like TestNG has

### DISCUSSION THREAD

**Discussion by neerajkumarlad** [webdriverio/webdriverio#8413] [Q&A]
Hi
I am looking to archive the grouping of test cases.. eg 'Sanity' , 'Regression'

Like in TestNG, has this feature

@Test (groups = { "Sanity", "Regression" })

**Comment by erwinheitzman** [comment]
I think that suites fulfill this role: https://webdriver.io/docs/organizingsuites/

**Reply by umarkhanTR** [reply]
How can I run all test suites in one command. Say I have 10 suites in my config file. Their names are:
testuser01
testuser02
...
testuser10
testuser11

I want to run all test suites in parallel.
I tried this command but did not work.

 npx wdio wdio.conf.js --suite testsuiteuser*

**Reply by christian-bromann** [reply]
just define all values of all your test suites into a single test suite, e.g. `runAll`.

**Reply by umarkhanTR** [reply]
Thanks.

**Reply by umarkhanTR** [reply]
Hi @christian-bromann 

Thanks for the suggestion. In my individual test suites, I declared them as array so that they can run in sequence. But when I use that nested [[ format in runAll, it gives error.

https://github.com/webdriverio/webdriverio/issues/8438

 runAll: [
      [
        [
          "./test/specs/test1.spec.js",
          "./test/specs/test2.spec.js",
        ],
      ],
      [
        [
          "./test/specs/test3.spec.js",
          "./test/specs/test4.spec.js",
        ],
      ],
    ],
